bodhi/tests/server/services/test_updates.py

```diff
@@ -2238,6 +2429,8 @@ def test_request_to_stable_based_on_stable_karma(self, *args):
         up = Update.get(nvr, self.db)
         up.status = UpdateStatus.testing
         up.request = None
+        eq_(len(up.builds), 1)
```

Comment:
I don't think we need this assertion or the similar ones below. This test module isn't a good example of the following ideology, but I believe that tests are best when they are as short as possible and test a condition with "laser" precision. A lot of the tests in this module are huge and are really hard to understand because they have a lot of extraneous code and extraneous assertions, which make it unclear what their purpose is.